frontend/packages/kubevirt-plugin/src/models/yaml-templates.ts

```diff
@@ -1,7 +1,7 @@
 import { Map as ImmutableMap } from 'immutable';
 
 import { TemplateModel } from '@console/internal/models';
-import { VirtualMachineModel } from './models';
+import { VirtualMachineModel } from '.';
 
 export const yamlTemplates = ImmutableMap()
```

Comment:
can we go one level deeper to `models/templates/` and have a separate file for each template?

Reply:
make sense to me too ... :+1: